revolv/static/accounting.scss

```diff
@@ -0,0 +1,115 @@
+body {
+    background-color: white;
+}
+
+.columns {
+    margin: 0;
+    padding: 0;
+}
+
+.heading {
+    .revolv-header-text {
+        font-size: 1.3rem;
+        font-weight: 800;
+        margin-top: 30px;
+    }
+    .filter-form {
+        .form-column {
+            margin: 0;
+        }
+
+        input, select {
+            width: 80%;
+        }
+
+        .field-label {
+            margin: 0;
+        }
+
+        .expense-link {
+
+            margin-top: 30px;
+
+            .link {
+                height: 100%;
+                width: 100%;
+            }
+        }
+    }
+}
+
+table, tr, th, td, tbody, thead {
+    border-collapse: collapse;
+    border-width: 0;
+    background-color: white;
+}
+
+.cash-flow-statement {
+
+    .cash-flow-header-text {
+        font-weight: 800;
+        font-size: 1.5rem;
+        margin: 0;
+    }
+    
+    .time-sub-text {
+        color: black;
+        font-style: italic;
+        margin: 0 0 10px 0;
+    }
+
+    .cash-flow-table {
+        width: 100%;
+        margin-bottom: 100px;
+
+        td, th {
+            padding: 3px;
+            font-size: 0.9rem;
+            font-weight: 300;
+        }
+
+        .payment-service-fees, .revolv-earnings {
+            color: red;
```

Comment:
This should be the red from revolv-settings - also, it doesn't seem to appear in the screenshot